eng/common/scripts/Verify-RestApiSpecLocation.ps1

```diff
@@ -182,39 +172,29 @@ function Verify-PackageVersion() {
         & $savePropertiesScriptPath -serviceDirectory $ServiceDirectory -outDirectory $PackageInfoDirectory
       }
     }
+    $pkgPropPath = Join-Path -Path $PackageInfoDirectory "$PackageName.json"
```

Comment:
PackageName is actually ArtifactName https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/stages/archetype-js-release.yml#L49

Reply:
I clarified the parameter description.